latex/dictionaries/forbidden-breaks

```diff
@@ -0,0 +1,127 @@
+กลโกธา
```

Comment:
@bountonw If you can check this list and let me know any changes or feel free to make whatever changes yourself.